R/spell-check.R

```diff
@@ -29,7 +29,10 @@
 #' @param pkg path to package root directory containing the `DESCRIPTION` file
 #' @param vignettes check all `rmd` and `rnw` files in the pkg root directory (e.g.
 #' `readme.md`) and package `vignettes` folder.
-#' @param use_wordlist ignore words in the package [WORDLIST][get_wordlist] file
+#' @param use_wordlist (logical or character)
```

Comment:
Instead of varying class `use_wordlist` I would add new argument `wordlist_path` and then pass it to the `get_wordlist` 
Extend `get_wordfile` `get_wordlist` and `update_wordlist` by this parameter. 
It would help also in some cases when one wants to have WORDLIST outside of the package - to have one WORDLIST for all tested packages.